receiver/hostmetricsreceiver/internal/scraper/diskscraper/disk_scraper_windows.go

```diff
@@ -121,6 +134,16 @@ func (s *scraper) Initialize(_ context.Context) error {
 		return err
 	}
 
+	s.avgDiskSecsPerReadCounter, err = pdh.NewPerfCounter(logicalAvgDiskSecsPerReadPath, true)
+	if err != nil {
+		return err
```

Comment:
do these errors need more context so we know which perf counter failed to be created or will the message returned from pdh.NewPerfCounter include the counter name?

Reply:
@james-bebbington please do this in a separate PR to keep the ball rolling :)

Reply:
You're right the errors can be generic such as "The specified counter could not be found". Will create a follow up PR to change this across the receiver.